files/build_templates/docker_image_ctl.j2

```diff
@@ -27,6 +27,21 @@ link_namespace() {
 }
 {%- endif %}
 
+function updateSyslogConf()
+{
+    # On multiNPU platforms, change the syslog target ip to docker0 ip to allow logs from containers
+    # running on the namespace to reach the rsyslog service running on the host
+    # Also update the container name  
+    if [[ ($NUM_ASIC -gt 1) ]]; then
+        TARGET_IP=$(ip -o -4 addr list docker0 | awk '{print $4}' | cut -d/ -f1)
```

Comment:
>TARGET_IP=$(ip -o -4 addr list docker0 | awk '{print $4}' | cut -d/ -f1) [](start = 8, length = 72)

This IP address is actually available in build time. Check `files/docker/docker.service.conf`

Reply:
Yes, I didn't want to hard code the ip address here. So using this method to get the docker0 ip address

Reply:
I mean `docker0` IP address is actually available in build time. Check `files/docker/docker.service.conf`. You are ok to use `ip` command, but build time constant may be easier.

---
In reply to: [439581391](https://github.com/Azure/sonic-buildimage/pull/4738#discussion_r439581391) [](ancestors = 439581391)

Reply:
I see. Your solution is future proof.

---
In reply to: [440542831](https://github.com/Azure/sonic-buildimage/pull/4738#discussion_r440542831) [](ancestors = 440542831,439581391)